subsys/bluetooth/controller/ll_sw/ull_adv.c

```diff
@@ -842,7 +842,14 @@ u8_t ll_adv_enable(u8_t enable)
 				   ticks_anchor, 0,
 				   HAL_TICKER_US_TO_TICKS((u64_t)interval *
 							  625),
-				   TICKER_NULL_REMAINDER, TICKER_NULL_LAZY,
+				   TICKER_NULL_REMAINDER,
+#if !defined(CONFIG_BT_TICKER_COMPATIBILITY_MODE) && \
+	!defined(CONFIG_SOC_COMPATIBLE_NRF)
+				   /* Force expiry to ensure timing update */
+				   TICKER_LAZY_MUST_EXPIRE,
```

Comment:
Just a reminder for me, is the TICKER_LAZY_MUST_EXPIRE supplied to the ticker_cb dynamic and used only when the event does not get the time reservation to use the radio?

Reply:
> Just a reminder for me, is the TICKER_LAZY_MUST_EXPIRE supplied to the ticker_cb dynamic and used only when the event does not get the time reservation to use the radio?

Yes, that's exactly how it works